src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -90,7 +90,7 @@ sap.ui.define([
 			this.oDefaultUri = this.sDefaultFullUrlType === "url" ? new Url() : new Uuid();
 			this.iSizeLimit = 10;
 			if (mParameters && mParameters.simpleFiltering === false){
```

Comment:
do we already have introduced the "simpleFiltering" option? I was not aware of that :-)

Reply:
changed the key to as per suggestion.